scripts/tasks/api-extractor.ts

```diff
@@ -14,10 +16,79 @@ export function apiExtractor() {
   return apiExtractorConfigs.length
     ? series(
         ...apiExtractorConfigs.map(([configPath, configName]) => {
+          // note we have `"strictNullChecks": false,` turned off - so from TS point of view this always returns API 🚨
+          const overrideApi = overrideExtractorConfigForPackagesWithTsPathAliases({
+            apiExtractorConfigPath: configPath,
+            tsConfigPath: resolveCwd('./tsconfig.json'),
+          });
+
           const taskName = `api-extractor:${configName}`;
-          task(taskName, apiExtractorVerifyTask({ configJsonFilePath: configPath, localBuild }));
+
+          task(
+            taskName,
+            series(
+              () => {
+                overrideApi && overrideApi.overrideConfig();
+              },
+              apiExtractorVerifyTask({ configJsonFilePath: configPath, localBuild }),
+              () => {
+                overrideApi && overrideApi.resetConfig();
+              },
+            ),
+          );
+
           return taskName;
         }),
       )
     : 'no-op';
 }
+
+interface TsConfig {
+  extends?: string;
+
+  /**
+   * typescript doesn't provide a correct type for the compiler options file
+   * -> (typescript.CompilerOptions has enum values instead of raw options in some cases)
+   */
+  compilerOptions: TscTaskOptions;
+  include?: string[];
+  exclude?: string[];
+}
+
+function overrideExtractorConfigForPackagesWithTsPathAliases(options: {
+  tsConfigPath: string;
+  apiExtractorConfigPath: string;
+}) {
+  const tsConfig: TsConfig = jju.parse(fs.readFileSync(options.tsConfigPath, 'utf-8'));
+  const overriddenTsConfig = { ...tsConfig };
```

Comment:
nit, I'm not sure there's much point in copying the object here since it's not a deep copy (when modifying `compilerOptions` below you're actually modifying the original)

Reply:
true that, just muscle memory. I dislike mutating live bindings (most of the time) so I always try to go immutable/functional way ...